tools/cloud-build/daily-tests/blueprints/crd-ubuntu.yaml

```diff
@@ -0,0 +1,37 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+
+blueprint_name: crd-ubuntu
+
+deployment_groups:
+- group: primary
+  modules:
+  - id: network
+    source: modules/network/vpc
+
+  - id: crd
+    source: community/modules/remote-desktop/chrome-remote-desktop
+    use: [network]
+    settings:
+      install_nvidia_driver: true  # Must be set to true to accept Nvidia ToS for driver install
+      instance_image:
```

Comment:
nit: you could achieve the same default/ubuntu behavior with a single blueprint that does not define instance_image and then have one test pass in instance_image on the command line.

I just wanted to point it out incase you had not already thought of that. I am kind of indifferent as the way you have it is not that much duplication and is more explicit. So I would be fine either way.

Reply:
acknowledged, frankly I was thinking about ways to do it but didn't come up with this solution.
Though it's good solution, I don't like the idea of using variables not declared in the blueprint.